### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**

### CODE REVIEW

**Review by Yeliseiev — APPROVED** [review]
Good job! ᕙ⁠(⁠⇀⁠‸⁠↼⁠‶⁠)⁠ᕗ

File: src/robotFactory.js [hunk 1]
```diff
@@ -1,10 +1,74 @@
...
+  };
+
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  };
+
+  getInfo() {
+    return `Robot: ${this.name}, `
++ `Chip version: ${this.chipVersion}, Weight: ${this.weight}`;
+  }
```
Comment (naziamoff) [inline src/robotFactory.js:34]: let's practice some destructuring here

File: src/robotFactory.js [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+
+  goDown(step = 1) {
+    this.coords.z -= step;
+  }
+}
+
+class DeliveryDrone extends FlyingRobot {
+  constructor(name, weight,
+    coords, chipVersion, maxLoadWeight, currentLoad = null) {
```
Comment (naziamoff) [inline src/robotFactory.js:55]: think of a way to make it look more neat, that's not good if you break line like that

**Review by naziamoff — CHANGES_REQUESTED** [review]
Well done! Just two small comments

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**